requirements/edx/github.txt

```diff
@@ -40,7 +40,7 @@ git+https://github.com/edx/rfc6266.git@v0.0.5-edx#egg=rfc6266==0.0.5-edx
 -e git+https://github.com/edx/event-tracking.git@0.2.0#egg=event-tracking
 -e git+https://github.com/edx-solutions/django-splash.git@7579d052afcf474ece1239153cffe1c89935bc4f#egg=django-splash
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@release-2015-07-14T11.10#egg=edx-ora2
+-e git+https://github.com/edx/edx-ora2.git@clintonb/drf-upgrade#egg=edx-ora2==0.0.1
```

Comment:
I could; but, that would require making more changes to either edx-ora2 or edx-platform than I care to make. I want to touch as little of this code as possible. I like my ecommerce ivory tower and very much wish to return to it.